app/package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "nftoftheday",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
hey could you update to the lastest node + npm please. it will stop this happening every time